project-reports/2024/2024-Q3-Hyperledger-Cacti.md

```diff
@@ -0,0 +1,95 @@
+---
+layout: default
+title: 2024 Q3 Hyperledger Cacti
+parent: 2024
+grand_parent: Project Updates
+---
+
+
+# Project Health
+
+- The project is in good health!
+- Multiple mentorship projects are underway, namely [Enabling Enterprise Organizational Blockchain Interoperability](https://wiki.hyperledger.org/display/INTERN/Enabling+Enterprise+Organizational+Blockchain+Interoperability) and [Cacti: Core Operators Modules for DLTs](https://wiki.hyperledger.org/display/INTERN/Cacti%3A+Core+Operators+Modules+for+DLTs).
+- `v2.0.0-rc.2` was released a couple weeks ago and `v2.0.0-rc.3` is not far behind (hopefully next week)!
+- Major efforts are underway to integrate the codebase further
+- We continue to promote interoperability protocol standardization by contributing to, and implementing, the candidate [IETF SATP specification](https://datatracker.ietf.org/doc/draft-ietf-satp-core/) using different combinations of Cacti modules. We conducted a [workshop on May 23, 2024](https://wiki.hyperledger.org/display/events/Standardizing+DLT+Interoperation%3A+Implementing+IETF+Secure+Asset+Transfer+Protocol+in+Hyperledger+Cacti) to discuss the protocol and demonstrate the associated Cacti capabilities.
+- Enabled the [OpenSSF Scorecard](https://scorecard.dev/viewer/?uri=github.com%2Fhyperledger%2Fcacti) for the project.
```

Comment:
[comments]
1. +1
2. Does the project team have a plan to triage and/or mitigate the vulnerabilities called out in the scorecard?

Reply:
@arsulegai Yes, the biggest item we have is that for some reason it doesn't recognize the DAST we have in place through our GitHub actions (we use nuclei)